### PR TITLE
Add recipe for Flac (needs libogg recipe)

### PR DESCRIPTION
**Pull request by dschreij**

### CODE REVIEW

File: recipes/flac/build.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+./autogen.sh
+./configure --prefix=${PREFIX} --enable-sse
```
Comment (jakirkham) [inline recipes/flac/build.sh]: Please add `make`. Also is there a `make check` we can add?

File: recipes/flac/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,175 @@
+cmake_minimum_required (VERSION 3.1.0)
```
Comment (jakirkham) [inline recipes/flac/CMakeLists.txt]: Please propose this upstream so they can have CMake support as well.

File: recipes/flac/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    - cmake  # [win]
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py35]
+    - pkgconfig  # [unix]
+    - autoconf  # [unix]
+    - automake  # [unix]
+    - libtool  # [unix]
+    - gettext  # [unix]
+    - ogg
```
Comment (jakirkham) [inline recipes/flac/meta.yaml]: Should we be pinning `ogg` somehow?

File: recipes/flac/build.sh (hunk 1 above)
Comment (dschreij) [inline recipes/flac/build.sh]: For the other xiph libs there were, so probably there are tests. I will add `make check`

File: recipes/flac/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  run:
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py35]
+    - ogg
+
```
Comment (dschreij) [inline recipes/flac/meta.yaml]: Yes, I'll add the file check tests again, but didn't get to that yet. Working on it now. Additionally, on Unix the make check command seems to do some functional tests of the library already.

**Comment by dschreij** [comment]
Closing because this has become a mess, and reopened at https://github.com/conda-forge/staged-recipes/pull/2101

File: recipes/flac/CMakeLists.txt (hunk 2 above)
Comment (dschreij) [inline recipes/flac/CMakeLists.txt]: I could do this, but my work is largely based on work by someone else: https://github.com/soulik/SDL_mixer/blob/master/external/flac-1.3.1/CMakeLists.txt and I don't think it is 'proper' if I submit this as a PR. I have contacted the author of the original if he is willing to do so, or if he is ok if I do it on his behalf.

Comment (dschreij) [inline recipes/flac/CMakeLists.txt]: Done at https://github.com/xiph/flac/pull/21

File: recipes/flac/meta.yaml (hunk 3 above)
Comment (jakirkham) [inline recipes/flac/meta.yaml]: Whatever you think is best. 👍

Would add a PR to update the [pinning script]( https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py ) with this and any other pinnings you have been adding/using.

**Pull request closed without merging**